### DISCUSSION TITLE
[Feature request] Disabling games closing after detecting bad connection

### DISCUSSION THREAD

**Discussion by Gajusz312** [ChristopherKlay/StadiaEnhanced#116] [Ideas & Requests]
Hi. I was wondering if there is a possibility you add feature stopping stadia from throwing me back onto home screen when it has detected bad connection. I am technically able to play on the platform with more than acceptable visual and no noticable input lag, but my internet connection is unstable at times, and I can't play for more than 10 minutes or so, which renders the service unplayable for me. Would be nice to be able to do something about it.
Thanks in advance.

**Comment by ChristopherKlay** [comment]
So far, i haven't found a way to stop Stadia from doing so and i don't believe it's possible either, since Stadia closes the current instance.
